zastosowania-komputerow-w-elektrotechnice.csl

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Zastosowania Komputerów w Elektrotechnice</title>
+    <title-short>ZKwE</title-short>
+    <id>http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice</id>
+    <link href="http://www.zotero.org/styles/zastosowania-komputerow-w-elektrotechnice" rel="self"/>
+    <link href="http://www.zotero.org/styles/ieee" rel="template"/>
+    <link href="http://www.iee.put.poznan.pl/caiee/redakcja.doc" rel="documentation"/>
+    <author>
+      <name>Mateusz Pacek</name>
+      <email>mateusz.m.pacek@student.put.poznan.pl</email>
+    </author>
+    <contributor>
+      <name>Michael Berkowitz</name>
+      <email>mberkowi@gmu.edu</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <category field="generic-base"/>
+    <updated>2015-04-13T10:05:52+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="pl-PL">
```

Comment:
this seems a bit odd: you're defining a couple of terms for Polish, where everything else (things like editor, e.g.) will be in English. What's the expectation for this style? Are papers going to be in Polish, English, or both and if both, does the citation style differ?